examples/trade_execution.py

```diff
@@ -1,16 +1,43 @@
+from examples.utils.consts import MarketIds
 from web3 import Web3
 from decimal import *
-import argparse
-from examples.utils.trade import execute_trade, getConfigs, MarketIds
+from examples.utils.trade import execute_trade, getConfigs
 
+def get_sample_signed_payloads(chain_id):
+    if chain_id == 1729:
+        return [{
+            'oraclePubKey': '0x51aa9e9C781F85a2C0636A835EB80114c4553098',
+            'pricePayload': {
+                'assetPairId': 'SOLUSDMARK',
+                'timestamp': '1731512244875654000',  # in MS * 10 ^ 6
+                'price': '215003440749999000000',  # WAD precision
+            },
+            # EIP712 signature of the price data
+            'r': '0x3820594933f6d003885a51bfc62b13f0518edae89507d6e89a022cf36975c49f',
+            's': '0x3c45a271ca6164a62f8b91ff21cf22d36399f5e406e8e8bbbcfdf799f905243e',
+            'v': 28,
+        }]
+
+    return [{
```

Comment:
I suppose this is testnet? we're not using SOLUSDMARK ticker there?

Reply:
yes